accesspoc/dips/apps.py

```diff
@@ -3,3 +3,6 @@
 
 class DipsConfig(AppConfig):
     name = 'dips'
+
+    def ready(self):
+        import dips.signals.handlers  # noqa F401
```

Comment:
Hi @sevein, I was "following" the Django docs:

https://docs.djangoproject.com/en/2.0/topics/signals/#connecting-receiver-functions

But you're right it looks excessive.

Reply:
Actually, I think it's not that bad.

I have added `dispatch_uid` to the receivers to avoid duplication in tests, although it's functionality is "deactivated" in them to avoid ES request.

Please, let me know if you really think we should move them out of.

Reply:
From the link you've shared:

> In practice, signal handlers are usually defined in a signals submodule of the application they relate to.

But I think it's ultimately up to you. We have in SS a package of models where each submodule is dedicated to a model and I like it.

I was looking at django-oscar:

![image](https://user-images.githubusercontent.com/606459/41473207-73c7e120-706d-11e8-8cc3-9fe8d89f144e.png)